src/components/index.js

```diff
@@ -93,6 +93,7 @@ export Messages from '@plone/volto/components/manage/Messages/Messages';
 export BlockChooser from '@plone/volto/components/manage/BlockChooser/BlockChooser';
 export Toolbar from '@plone/volto/components/manage/Toolbar/Toolbar';
 export Sidebar from '@plone/volto/components/manage/Sidebar/Sidebar';
+export SidebarPopup from '@plone/volto/components/manage/Sidebar/SidebarPopup';
```

Comment:
I'm not sure we want this line. Would webpack drop this component from the bundle if we don't do this? In theory only addons that use this component would activate the inclusion of the module in the bundle.